runtime/src/lib.rs

```diff
@@ -102,13 +103,15 @@ fn enter_idle(drivers: &mut Drivers) {
                 Err(e) => caliptra_drivers::report_fw_error_non_fatal(e.into()),
             }
         }
+        // Don't enter low power mode when in progress
+        return;
```

Comment:
If we enter the `Ok` case on line 102, I think we do want to enter a sleep state (because our in-progress operation was completed). In that case we don't need to do anything else until the caller calls `GET_RESULTS`.

So I think we just want this return if the case where `drivers.mbox.lock() == true` (the else case).

Reply:
Done